addon/components/o-s-s/currency-input.ts

```diff
@@ -8,6 +8,7 @@ interface OSSCurrencyInputArgs {
   currency: string;
   value: number;
   onChange(currency: string, value: number): void;
+  onlyCurrency: boolean;
```

Comment:
`currencyOnly?: boolean`.

In Typescript, the `?` at the end means that the parameter can be undefined and is a shortcut for:
- `onlyCurrency: boolean | undefined`